### DISCUSSION TITLE
Welcome to STAC Discussions!

### DISCUSSION THREAD

**Discussion by cholmes** [radiantearth/stac-spec#1141] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. Feel free to discuss anything about STAC spec itself, its [extensions](https://github.com/stac-extensions/), or any of the wider ecosystem of tools (many available at [stac-utils](https://github.com/stac-utils/stactools)). We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gadomski** [comment]
👋🏽 working primarily on [stactools](https://github.com/stac-utils/stactools), also helping a bit with [pystac](https://github.com/stac-utils/pystac) and [pystac-client](https://github.com/stac-utils/pystac-client).

**Comment by joshfix** [comment]
Hi all -- currently maintaining Staccato: [[github](https://github.com/planetlabs/staccato)] [[demo deployment](https://staccato.space)]

**Comment by duckontheweb** [comment]
Hi all! I'm with [Radiant Earth Foundation](https://www.radiant.earth/) and relatively new to the STAC community. I'm working mostly on [pystac](https://github.com/stac-utils/pystac), with some contributions [pystac-client](https://github.com/stac-utils/pystac-client) as well. I'm also maintaining the [radiant-mlhub](https://github.com/radiantearth/radiant-mlhub) Python client, which is a PySTAC-based client for working with our Radiant MLHub API.

**Comment by jjrom** [comment]
Hi all !
Maintaining resto server [[github](https://github.com/jjrom/resto)] [[Demo deployment](https://tamn.snapplanet.io)] and [[rocket](https://rocket.snapplanet.io)] web client

**Comment by m-mohr** [comment]
Hi all,

I started with STAC during the third sprint, around STAC 0.6.

I'm Technical Fellow with Radiant Earth and as such working on
- STAC Browser (+ stac-fields)
- stac-migrate
- STAC Node Validator
- STAC specification (incl. JSON Schemas)
- STAC Index

I'm also working for openEO and as such working on:
- a bit of STAC API
- integrating STAC into openEO
- implementing STAC API for the Google Earth Engine driver in openEO
- numerous extensions, e.g. datacube, card4l, file, sar, ...

**Comment by fredliporace** [comment]
Hello!

I'm CTO at AMS Kepler, we develop ground station software for EO satellites. I manage [CBERS on AWS](https://registry.opendata.aws/cbers/) public dataset, including [CBERS on AWS STAC service](https://github.com/fredliporace/cbers-2-stac)

**Comment by jbants** [comment]
Hi All,
I'm with [Sparkgeo](www.sparkgeo.ca) and help maintain [stac-validator](https://github.com/sparkgeo/stac-validator) and [stac-lint](https://staclint.com/).

Helping out with [stactools](https://github.com/stac-utils/stactools).

**Comment by constantinius** [comment]
Hi folks!

I'm with [EOX](http://eox.at) and mainly a user of STAC. I'm also the main author of the rather esoteric extension [tiled-assets](https://github.com/stac-extensions/tiled-assets)

**Comment by lossyrob** [comment]
Hi! I'm working at Microsoft on the [Planetary Computer](https://planetarycomputer.microsoft.com/), focused on ingest pipelines that utilize [stactools](https://github.com/stac-utils/stactools) and an API that uses [stac-fastapi](https://github.com/stac-utils/stac-fastapi). I'm the original author of [pystac](https://github.com/stac-utils/pystac).
